.github/workflows/ci.yml

```diff
@@ -89,6 +110,19 @@ jobs:
           path: ${{ github.workspace }}/cover.html
         if: always()
 
+  njs-unit-tests:
```

Comment:
Because `unit-tests` Makefile target depends on `njs-unit-test`, the  action `unit-tests` will run njs unit tests:
```
      - name: Run Tests
        run: make unit-test
 ```
so we will have two runs, right?

Reply:
Nice catch, I missed that. I'd like the NJS tests to run separately to avoid running a docker container in the github runners. I'll remove `njs-unit-test` from the `unit-tests` target and add it to `dev-all`.